tidb-specific-system-variables.md

```diff
@@ -578,18 +591,145 @@ set tidb_query_log_max_len = 20
 - Default value: 0
 - This variable is used to show whether the execution plan used in the previous `execute` statement is taken directly from the plan cache.
 
+### ddl_slow_threshold
+
+- Scope:SESSION
+- Default value:300
+- Ddl operations will be output to the log if its time consumption exceeds the threshold value. The unit is millisecond.
+
+### tidb_pprof_sql_cpu <span class="version-mark">New in v4.0</span>
+
+- Scope:SESSION
+- Default value:0
+- This variable is used to control whether to mark the corresponding SQL statement in the profile output to identify and troubleshoot performance issues.
+
+### tidb_skip_isolation_level_check
+
+- Scope:SESSION
+- Default value:0
+- After this switch is enabled, if an isolation level not supported by TiDB is assigned to `tx_isolation`, no error will be reported.
+
+### tidb_low_resolution_tso
+
+- Scope:SESSION
+- Default value:0
+- This variable is used to set whether to enable the low precision tso feature. After this feature is enabled, new transactions will use a ts updated every 2s to read data
+- The main scenario is to reduce the overhead of acquiring tso for small read-only transactions when old data can be tolerated.
+
+### tidb_replica_read <span class="version-mark">New in v4.0</span>
+
+- Scope:SESSION
+- Default value: leader
+- This variable is used to control where TiDB reads data. Here are three options:
+    - leader: Read only from leader node
+    - follower: Read only from follower node
+    - leader-and-follower: Read from leader or follower node
+
+### tidb_use_plan_baselines <span class="version-mark">New in v4.0</span>
+
+- Scope:SESSION | GLOBAL
+- Default value: on
+- This variable is used to control whether to enable the execution plan binding feature. It is enabled by default, and can be disabled by assigning the value off. For the use of the execution plan binding, see [Execution Plan Binding](/execution-plan-binding.md#创建绑定).
+
+### tidb_capture_plan_baselines <span class="version-mark">New in v4.0</span>
+
+- Scope:SESSION | GLOBAL
+- Default value: off
+- This variable is used to control whether to enable the automatic binding feature. This feature depends on the Statement Summary, so you need to turn on the Statement Summary before using automatic binding.
+- After this feature is enabled, the historical SQL statements in the Statement Summary will be traversed periodically, and bindings will be automatically created for SQL statements that appear at least twice.
+
+### tidb_evolve_plan_baselines <span class="version-mark">New in v4.0</span>
+
+- Scope:SESSION | GLOBAL
+- Default value: off
+- This variable is used to control whether to enable the baseline evolution feature. For more details, see [Baseline Evolution](/execution-plan-binding.md#自动演进绑定).
```

Comment:
```suggestion
- This variable is used to control whether to enable the baseline evolution feature. For detailed introduction or usage , see [Baseline Evolution](/sql-plan-management.md#baseline-evolution).
```
When translating the Chinese PR, you have to use an English link anchor too rather than `#自动演进绑定`.